src/profile-logic/marker-styles.js

```diff
@@ -53,11 +53,18 @@ export const markerStyles: MarkerStyles = {
     height: 18,
     squareCorners: true,
   },
+  // Scripts is renamed to 'requestAnimationFrame callbacks' but keeping this
+  // here for backwards compatibility.
```

Comment:
I think we should do a profile upgrade instead so that we don't need this backwards compatibility, what do you think? Also this way we won't conflict if we reuse "Scripts" markers in the future for other things.
It's less important for TTFI and other less common names, but maybe we should do it too?

Reply:
It would be more disciplined to do an upgrader, but the impact is minimal I think if we I just ignore it, and move on. So I would be fine with either decision.